build.gradle

```diff
@@ -589,6 +589,9 @@ project('spring-integration-debezium') {
         }
 
         testImplementation "io.debezium:debezium-connector-mysql:$debeziumVersion"
+        testImplementation "io.debezium:debezium-connector-postgres:$debeziumVersion"
+        testImplementation "io.debezium:debezium-connector-sqlserver:$debeziumVersion"
```

Comment:
This is what I meant about Debezium BOM: https://repo1.maven.org/maven2/io/debezium/debezium-bom/2.3.0.CR1/debezium-bom-2.3.0.CR1.pom